src/pages/Compare/index.tsx

```diff
@@ -179,7 +179,10 @@ export class CompareBase extends React.Component<Props> {
         );
       }
 
-      if (!versionFileIsLoading && versionFile === undefined) {
+      const versionFileIsReady =
```

Comment:
I like the logic, but I'm not sure about the variable name. I was initially confused reading this because I couldn't understand why `entryStatusMap[version.selectedPath] === 'D'` meant that the versionFile was "Ready". What you're really saying is that we don't need to fetch a version file if we either already have one, or `entryStatusMap[version.selectedPath] === 'D'`, so maybe this could be called `shouldFetchVersionFile`, and reverse the values?

Reply:
`shouldFetchVersionFile` is perfect! Thanks! I struggled a lot to find a proper name for this variable.